README.md

```diff
@@ -108,6 +108,32 @@ module.exports = {
 }
 ```
 
+### Disabling HMR
+
+By default Hot Module Replacement is disabled in following situations:
+
+ * Webpack `target` is `node`
+ * Webpack doesn't minify the code 
+ * `process.env.NODE_ENV !== 'production'`
```

Comment:
I need to go home now :(

What is the status? How may I help (more clear code for handling HMR conditions, unit-tests) ?